packages/medusa/src/api/routes/admin/price-lists/index.ts

```diff
@@ -93,25 +93,143 @@ export const defaultAdminPriceListFields = [
+ *     description: The type of the object that was deleted.
+ *     default: money-amount
+ *   deleted:
+ *     type: boolean
+ *     description: Whether or not the items were deleted.
+ *     default: true
+ */
 export type AdminPriceListDeleteBatchRes = {
   ids: string[]
   deleted: boolean
   object: string
 }
 
+/**
+ * @schema AdminPriceListDeleteProductPricesRes
+ * type: object
+ * properties:
+ *    ids:
+ *     type: array
+ *     description: The price ids that have been deleted.
+ *     items:
+ *       type: string
+ *    object:
+ *      type: string
+ *      description: The type of the object that was deleted.
+ *      default: money-amount
+ *    deleted:
+ *      type: boolean
+ *      description: Whether or not the items were deleted.
+ *      default: true
+ */
 export type AdminPriceListDeleteProductPricesRes = AdminPriceListDeleteBatchRes
+
+/**
+ * @schema AdminPriceListDeleteVariantPricesRes
+ * type: object
+ * properties:
+ *    ids:
+ *     type: array
+ *     description: The price ids that have been deleted.
+ *     items:
+ *       type: string
+ *    object:
+ *      type: string
+ *      description: The type of the object that was deleted.
+ *      default: money-amount
+ *    deleted:
+ *      type: boolean
+ *      description: Whether or not the items were deleted.
+ *      default: true
+ */
 export type AdminPriceListDeleteVariantPricesRes = AdminPriceListDeleteBatchRes
 
+/**
+ * @schema AdminPriceListDeleteRes
+ * type: object
+ * properties:
+ *   id:
+ *     type: string
+ *     description: The ID of the deleted Price List.
+ *   object:
+ *     type: string
+ *     description: The type of the object that was deleted.
+ *     default: price-list
+ *   deleted:
+ *     type: boolean
+ *     description: Whether or not the items were deleted.
+ *     default: true
+ */
 export type AdminPriceListDeleteRes = DeleteResponse
 
+/**
+ * @schema AdminPriceListsListRes
+ * type: object
+ * properties:
+ *   price_lists:
+ *    type: array
+ *    items:
+ *      $ref: "#/components/schemas/PriceList"
+ *   count:
+ *     type: integer
+ *     description: The total number of items available
+ *   offset:
+ *     type: integer
+ *     description: The number of items skipped before these items
+ *   limit:
+ *     type: integer
+ *     description: The number of items per page
+ */
 export type AdminPriceListsListRes = PaginatedResponse & {
   price_lists: PriceList[]
 }
 
+/**
+ * @schema AdminPriceListsProductsListRes
+ * type: object
+ * properties:
+ *   products:
+ *     type: array
+ *     items:
+ *       $ref: "#/components/schemas/Product"
+ *   count:
+ *     type: integer
+ *     description: The total number of items available
+ *   offset:
+ *     type: integer
+ *     description: The number of items skipped before these items
+ *   limit:
+ *     type: integer
+ *     description: The number of items per page
+ */
+export type AdminPriceListsProductsListRes = PaginatedResponse & {
```

Comment:
Missing declaration for `admin/price-lists/list-price-list-products.ts`. 
JS client has response typed as `any`.  [See file](https://github.com/medusajs/medusa/blob/82ea9c4ea8e147504844b0b995c0a791bd17b783/packages/medusa-js/src/resources/admin/price-lists.ts#L65-L78).